### DISCUSSION TITLE
Would it be valuable to add TA-Lib integration?

### DISCUSSION THREAD

**Discussion by crazy25000** [nautechsystems/nautilus_trader#387] [Ideas]
This wrapper is written in Cython and figured it's appropriate for this project: https://mrjbq7.github.io/ta-lib/index.html

>         Includes 150+ indicators such as ADX, MACD, RSI, Stochastic, Bollinger Bands, etc.
>         Candlestick pattern recognition

I have a few working examples and can open a PR if we want to integrate them. Would add automatic support for common indicators + candle stick patterns.

**Comment by cjdsellers** [comment]
Sorry for taking so long to see this.

TA-Lib has been brought up by others, I think it would definitely be a valuable addition.

Opening an issue to track progress.

**Comment by rterbush** [comment]
FWIW, there are several Rust wrappers out there as well.

https://github.com/CLevasseur/ta-lib-rust

**Comment by rsmb7z** [comment]
Now supported.
